go-runtime/compile/schema.go

```diff
@@ -771,10 +780,10 @@ func visitType(pctx *parseContext, pos token.Pos, tnode types.Type) (schema.Type
 		if underlying.String() == "any" {
 			return &schema.Any{Pos: goPosToSchemaPos(pos)}, nil
 		}
-		return nil, tokenErrorf(pos, tnode.String(), "unsupported type %q", tnode)
+		return nil, tokenErrorf(pos, "", "unsupported type %q", tnode)
```

Comment:
Why'd this change?

Reply:
I'd love a better approach for this, but I wasn't sure how to get the right info here. `pos` is the position of the field name afaict. and `tnode` is the type. So, it seems like I can only get the pos from the field name and not the type. Which left me with a starting `pos` for the field name and a `EndColumn` from the type name. (I hope that sentence isn't too hard to read, it's breaking my brain).

So instead I use the pos from the field name and make the `EndColumn` match, which will trigger the file parsing in the LSP.

Would love to have a better solution here though

Reply:
makes sense